src/test/java/com/znsio/api/fetchPostsAndAlbums.feature

```diff
@@ -0,0 +1,36 @@
+@UserPosts @prod
+Feature: Fetch User Posts and Albums
+
+  Background:
+    * def userId = 1
+    * def testData = read('classpath:com/znsio/templates/userData.json')
+
+  @getPosts
+  Scenario: Get posts with specific User id
+    * def fetchPosts = karate.call('classpath:com/znsio/templates/fetchPostAndAlbums.feature@t_getPost', {"userId": userId, expectedStatus: 200})
+    * karate.log('response : ',fetchPosts.response)
+    * match each fetchPosts.response[*].userId == userId
+    * match fetchPosts.response.userId == '#number'
+    * match fetchPosts.response.id == '#number'
+    * match fetchPosts.response.title == '#string'
+    * match fetchPosts.response.body == '#string'
+
+
+  @getPosts
+  Scenario: Get posts with user id that doesn't exist
+    * def fetchPosts = karate.call('classpath:com/znsio/templates/fetchPostAndAlbums.feature@t_getPost', {"userId": 100, expectedStatus: 200})
+    * karate.log('response : ',fetchPosts.response)
+    * assert fetchPosts.response.arr == null
+
```

Comment:
Try scenario outline  and add few more invalid cases.

Reply:
added